app/controllers/admin/organisations_controller.rb

```diff
@@ -22,14 +23,25 @@ def destroy
 private
 
   def sortable_columns
-    %w[name created_at active_storage_attachments.created_at]
+    %w[name created_at active_storage_attachments.created_at address]
   end
 
   def sort_column
-    sortable_columns.include?(params[:sort]) ? params[:sort] : "created_at"
+    sortable_columns.include?(params[:sort]) ? params[:sort] : sortable_columns[1]
```

Comment:
would be better to leave this one hard coded, as going for the second item is less intuitive than going for the first.

Reply:
Was trying to stay consistent but you make a valid point

Reply:
I made the change as it was more readable imo